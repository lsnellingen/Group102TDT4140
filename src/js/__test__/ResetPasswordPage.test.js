import React from 'react';
import ReactDOM from 'react-dom';
import ResetPasswordPage from '../pages/ResetPasswordPage';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';

describe('ResetPasswordPage', () => {
  var ReactTestUtils = require('react-addons-test-utils') // ES5 with npm

    test('renders without crashing', () => {
      /*const renderer = ReactTestUtils.createRenderer();*/

      const tree = renderer.create(
        <ResetPasswordPage/>).toJSON();


      expect(tree).toMatchSnapshot();




    });
});
