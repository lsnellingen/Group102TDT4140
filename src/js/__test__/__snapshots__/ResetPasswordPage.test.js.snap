// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPasswordPage renders without crashing 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-xs-12"
    >
      <h3>
        Forgot Password
      </h3>
      <hr />
    </div>
  </div>
  <form
    onSubmit={[Function]}
  >
    <div>
      <div
        className="sp-reset-password-form"
      >
        <div
          className="row"
        >
          <span />
          <div
            className="col-xs-12"
            data-spIf="!form.sent"
          >
            <div
              className="form-horizontal"
            >
              <div
                className="form-group"
              >
                <label
                  className="col-xs-12 col-sm-4 control-label"
                  htmlFor="spEmail"
                >
                  Email or Username
                </label>
                <div
                  className="col-xs-12 col-sm-4"
                >
                  <input
                    className="form-control"
                    defaultValue=""
                    id="spEmail"
                    name="email"
                    onChange={[Function]}
                    placeholder="Your Email Address"
                  />
                </div>
              </div>
              <div
                className="form-group"
              >
                <div
                  className="col-sm-offset-4 col-xs-12"
                >
                  <span />
                  <button
                    className="btn btn-primary"
                    disabled={false}
                    type="submit"
                  >
                    Request Password Reset
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </form>
</div>
`;
